igvc_navigation/src/mapper/map_utils.h

```diff
@@ -62,6 +63,8 @@ struct GroundPlane
   double d;
 };
 
+class Ray;
```

Comment:
I'm forward declaring it here, since I need to use it in a function declaration below. `Ray` lives in `Octomapper`, which uses `MapUtils`, and so if ai just simply included then each would include each other, which doesn't work